test/e2e/performanceprofile/functests/utils/pods/pods.go

```diff
@@ -148,7 +148,7 @@ func ExecCommandOnPod(c *kubernetes.Clientset, pod *corev1.Pod, containerName st
 			Stdin:     false,
 			Stdout:    true,
 			Stderr:    true,
-			TTY:       true,
+			TTY:       false,
```

Comment:
uuuuuuhmmm
from the `client-go` docstrings:
```
	// StreamWithContext initiates the transport of the standard shell streams. It will
	// transport any non-nil stream to a remote system, and return an error if a problem
	// occurs. If tty is set, the stderr stream is not used (raw TTY manages stdout and
	// stderr over the stdout stream).
	// The context controls the entire lifetime of stream execution.
```

this can explain a lot of things

Reply:
Nice catch. This explains why we didn't face the issue with TuneD so far